Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -695,9 +695,29 @@ def test_log
         system "git", "commit", "--allow-empty", "-m", "This is a test commit"
       end
     end
+
+    core_tap = CoreTap.new
+    formula_file = core_tap.formula_dir/"testball.rb"
+    formula_file.write <<-EOS.undent
+      class Testball < Formula
+        url "https://example.com/testball-0.1.tar.gz"
+      end
+    EOS
+
+    FileUtils.cd "#{core_tap.path}".chomp do
```

Comment:
It's more concise and whenever possible we prefer to use `Pathname` and its methods when dealing with paths. Other than that, `FileUtils.cd core_tap.path` and `core_tap.path.cd` are pretty much equivalent.